sonarcloudtest/Controllers/NewController.cs

```diff
@@ -19,6 +19,7 @@ public IEnumerable<Ietm> Things()
             } else
             {
                 var error = "FAIL!";
+                var ehh = "ABE!!!";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'ehh' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1481)